src/fuse_api.pxi

```diff
@@ -313,6 +315,7 @@ def main(workers=None, handle_signals=True):
             session_loop_single()
         else:
             session_loop_mt(workers)
+    t.join()
```

Comment:
Would be a good idea to leave a comment here (in the code) on why this is needed, or someone may remove this again at some point as "not needed".